tests/cypress/integration/4-design-steps/header-menu.cy.js

```diff
@@ -49,15 +49,15 @@ describe( 'Header menu Page', function () {
 		} );
```

Comment:
In this I saw that now header menu takes time to load in the preview, should we also add a wait so that it gets rendered out as well?

Reply:
I meant after clicking on a new header pattern it renders very slowly.

Reply:
okay